dashboard/extract_s3.py

```diff
@@ -0,0 +1,55 @@
+"""
+Extracts the archived csv files from bucket 
+into the archived_data folder.
+"""
+import os
+from os import environ
+from os import path
+from dotenv import load_dotenv
+from boto3 import client
```

Comment:
don't think this is used in this file either, so can be removed?

Reply:
What do you mean by this?